src/nunit.analyzers/ConstActualValueUsage/ConstActualValueUsageCodeFix.cs

```diff
@@ -157,6 +155,32 @@ private static bool IsSupportedClassicAssert(IMethodSymbol methodSymbol)
                 SupportedClassicAsserts.Contains(methodSymbol.Name);
         }
 
+        private static bool IsSupportedIsConstraint(ref InvocationExpressionSyntax invocationExpression, [NotNullWhen(true)]out MemberAccessExpressionSyntax? memberAccessExpression)
+        {
+            if (invocationExpression.Expression is not MemberAccessExpressionSyntax mae)
+            {
+                memberAccessExpression = null;
+                return false;
+            }
+
+            memberAccessExpression = mae;
```

Comment:
I simplified this a tiny bit. Mainly because I don't like shorthand names like `mae`